src/Misc/layoutbin/en-US/strings.json

```diff
@@ -498,6 +498,7 @@
   "ResourceMonitorMemoryInfo": "Memory: Used {0} MB out of {1} MB",
   "ResourceMonitorMemoryInfoError": "Unable to get memory info, exception: {0}",
   "ResourceMonitorMemorySpaceIsLowerThanThreshold": "Free memory is lower than {0}%; Currently used: {1}%",
+  "ResourceUtilizationWarningsIsDisabled": "Resource Utilization warnings is disabled, switch \"DISABLE_RESOURCE_UTILIZATION_WARNINGS\" variable to \"true\" if you want to enable it",
```

Comment:
Isn't this backwards – one would set `DISABLE_RESOURCE_UTILIZATION_WARNINGS` to _false_ to enable resource utilization warnings?